showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITIam.java

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.showcase.v1beta1.it;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import com.google.api.gax.rpc.InvalidArgumentException;
+import com.google.api.gax.rpc.NotFoundException;
+import com.google.common.collect.ImmutableList;
+import com.google.iam.v1.Binding;
+import com.google.iam.v1.GetIamPolicyRequest;
+import com.google.iam.v1.Policy;
+import com.google.iam.v1.SetIamPolicyRequest;
+import com.google.iam.v1.TestIamPermissionsRequest;
+import com.google.iam.v1.TestIamPermissionsResponse;
+import com.google.showcase.v1beta1.IdentityClient;
+import com.google.showcase.v1beta1.it.util.TestClientInitializer;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITIam {
+  private static final Policy DEFAULT_POLICY =
+      Policy.newBuilder()
+          .addBindings(Binding.newBuilder().setRole("foo.editor").addMembers("allUsers"))
+          .build();
+  private static IdentityClient grpcClient;
+  private static IdentityClient httpjsonClient;
+  private String resourceName;
+
+  @BeforeClass
+  public static void createClients() throws Exception {
+    grpcClient = TestClientInitializer.createGrpcIdentityClient();
+    httpjsonClient = TestClientInitializer.createHttpJsonIdentityClient();
+  }
+
+  @Before
+  public void setupTests() {
+    resourceName = "users/" + UUID.randomUUID().toString().substring(0, 8);
```

Comment:
Why the resource name is in this format? I see that it is a wild card resource based on the definition in [googleapis](https://github.com/googleapis/googleapis/blob/bd3fbf4776fe1240f2fddcc525a6b6e7690232ad/google/iam/v1/iam_policy.proto#L106), I guess it is randomly chosen?

Reply:
I was using the IdentityClient to call the IAM RPCs which uses the User resource. I just gave it `user/{random_user_id}`.

Reply:
Discussed offline that this could be any format.